src/lib/db.ts

```
@@ -0,0 +1,164 @@
+  rate: number;
+}
+
+export interface ReleaseInfo {
+  id: number;
+  year: number;
+  month: number;
+  day: number;
+  event: string;
+}
+
+export async function getCards(limit: number = 50, offset: number = 0) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)